bigquery_etl/run_multipart_query.py

```diff
@@ -97,6 +97,7 @@ def dirpath(string):
     choices=[
         bigquery.SchemaUpdateOption.ALLOW_FIELD_ADDITION,
         bigquery.SchemaUpdateOption.ALLOW_FIELD_RELAXATION,
+        "",
```

Comment:
[telemetry-airflow passes an empty string here](https://github.com/mozilla/telemetry-airflow/blob/03ee091cdba5cb1fcb0a6aad3e62483d2bfbb3ee/dags/utils/gcp.py#L511)

Reply:
This would mean that we end up with `args.schema_update_options` being `[""]` which is passed to a `bigquery.QueryJobConfig` below. Do we know the behavior of that value in a job config?

Reply:
Based on the tests I ran locally, it would just ignore schema update options. But maybe the better solution here is to rewrite the part in telemetry-airflow where the schema_update_option is added as a parameter and not passing an empty string there.

Reply:
Given that fixing this on the Airflow side looks untenable, I think I'd be fine with this solution, with commentary indicating why we need to accept `""`.

Reply:
Added a comment. I'll leave this open for now until @relud had a chance to take a look, maybe there is a more elegant solution here.

Reply:
we could remove the empty string from the list in `main()`, given that it works with empty string in the list, this seems fine as-is